Source/Core/InputCommon/InputCommon.vcxproj

```diff
@@ -35,6 +35,16 @@
     <Import Project="..\..\VSProps\PCHUse.props" />
   </ImportGroup>
   <PropertyGroup Label="UserMacros" />
+  <ItemDefinitionGroup Condition="'$(Configuration)|$(Platform)'=='Release|x64'">
+    <ClCompile>
+      <AdditionalIncludeDirectories>C:\dev\opencv\build\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
I don't think we should be committing a local path like `C:\dev\opencv` here. Also, are those two `ItemDefinitionGroup` now obsolete that you removed the OpenCV dependency?

Reply:
Yes this whole block can be removed, thanks